scripts/setUpSite.js

```diff
@@ -1,7 +1,7 @@
-import { Map, TileLayer } from "leaflet";
+import { Map, TileLayer, CircleMarker } from "leaflet";
 import "leaflet/dist/leaflet.css";
-
 import setUpIcons from "../src/js/fontAwesome";
+import data from "../map/tidied_map_data.csv";
```

Comment:
You'll want to move this import into `setUpCityPointsLayer` using an `await import` like we do in PLM.